docs/file_explorer.md

```diff
@@ -1,68 +1,108 @@
 
-- **A.** Add a Workspace <br>
-- **B.** Rename a Workspace  <br>
-- **C.** Delete a Workspace  <br>
--  **D.** Download all Workspaces This will create a .zip file with all the files of all the Workspaces.  The zip file will have a folder called **.workspaces** that will contain a folder of each workspace.  Depending on your OS, you may need to change the preferences on .workspaces folder to make it visible.  <br>
-- **E.** Upload the Workspaces backup made from the previous icon. <br>
-- **F.** Clone a repo from Git repository <br>
-- **G.** Link to documentation <br>
-- **J.** Publish the Workspace to a GIST <br>
-- **L.** Choose a Workspace 
-### File Manipulation
-The letters in bold below refer to the labels in fig. 1.
+When choosing an Open Zeppelin template, additional functionality can be added.
+![](images/a-fe-modal-oz.png)
 
-- **H.** Create a file  <br>
-- **I.** Create a folder  <br>
-- **K.** Load a local file into the current Workspace<br>
+#### Workspace operations
 
-## Workspaces with Templates
-When you create a new Workspace, a modal comes up where you choose which template of files to include.
+The Workspace hamburger menu is for operations that work on an entire workspace.
 
-![](images/a-workspace-templates.png)
+![](images/a-fe-hamburger.png)
 
-## Workspaces & cloning a repo
-When you click the **clone** icon (**F.** in fig.1), you'll be asked for the url of the repo.  A new workspace will be created that will contain the cloned repo.  To manage the Git repo, go to the DGIT plugin.
+##### Clone
+When clicking Clone, you’ll be asked for the url of a remote repo. A new workspace will be created that will contain the cloned repo. To manage the Git repo, go to the DGIT plugin.
+##### Backup
+Backup is for downloading all the Workspaces in a .zip file. The zip file will have a folder called **.workspaces** that will contain a folder of each Workspace.  Depending on your OS, you may need to change the preferences on .workspaces folder to make it visible.  
 
-## Workspaces with a Git repo
+##### Restore
+Restore is only for uploading a .workspaces folder that was created using the Backup operation.
+
+##### Create GitHub Actions
+The Workspace operations to create **Solidity Test Workflow**,  **Moca Chai Test Workflow**, and **Slither Workflow** are for creating GitHub actions. When clicked, a .yml file in the .workflows folder of the active Workspace.
+
+### Workspaces initialized with Git
 Workspaces with an associated Git will have the Git icon next to them in the Workspaces select box.
 
 ![](images/a-fe-select-git.png)
 
+When creating a new Workspace, there is a checkbox in the modal for choosing the template that will initialize the Workspace for Git.
+
+Working with Files
+-------------------
+When a file is clicked on it will appear in the Editor.
+
+Files can be manipulated by right clicking on them ([see below](#right-click-on-a-file-or-folder)) and also by using the icons under the Workspace select box.
```

Comment:
`Files can be manipulated by right clicking on them` seems confusing to me

Reply:
updated